gitbootcamp.rst

```diff
@@ -194,7 +194,7 @@ Staging and Committing Files
 
 2. To stage the files to be included in your commit::
 
-      git add <filename1> <filename2>
+      git add -p <filename1> <filename2>
```

Comment:
Maybe we could suggest both.  Generally I just use `git add -p` (with no extra arguments), since it allows me to review all changes made to files before adding them, and `git add <filename>` to add new files.
```suggestion
      git add -p  # to review and add changes to existing files
      git add <filename1> <filename2>  # to add new files
```